topics/event-classes.md

```diff
@@ -6,32 +6,67 @@ body_class: event-classes
-| `resource.edit` | A resource was updated by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#resource-edit)  | <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i> |
 
-&#10033; These events are only available when using [New List View](https://support.talis.com/hc/en-us/articles/360000008189).
+## Talis Elevate
+
+| Class | Description | Timeseries data? | Raw data? |
+| :- | :- | :- | :- |
+| `player.timer.2` | A resource was viewed by a student in the player for an amount of time |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-timer-2)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.timer.2.contributors` | A resource was viewed by a contributor (academic, teaching assistant) in the player for an amount of time |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-timer-2-contributors)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.view` | A resource was opened in the player by a student |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-view)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.view.contributors` | A resource was opened in the player by a contributor (academic, teaching assistant) |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-view-contributors)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.annotate.createannotation` | A new annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.annotate.createreply` | A new replay (to an existing thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createreply)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.download` | The player resource was downloaded by a user | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-download)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```

Comment:
I believe we're missing `player.print`

Reply:
(Or does this not make it into Census?)

Reply:
Also, are we choosing to ignore the `.contributors` events?

Reply:
What are the `.contributors` events? I see some for some of the events...
What do they do and how are they used?
Actually maybe that is a question for another time.